library/std/math.qs

```diff
@@ -102,42 +101,36 @@ namespace Microsoft.Quantum.Math {
 
     /// # Summary
     /// Returns -1, 0 or +1 that indicates the sign of a number.
-    function SignI (a : Int) : Int {
-        if   (a < 0) { -1 }
-        elif (a > 0) { +1 }
-        else         { 0 }
+    function SignI(a : Int) : Int {
+        if (a < 0) { -1 } elif (a > 0) { + 1 } else { 0 }
     }
 
     /// # Summary
     /// Returns -1, 0 or +1 that indicates the sign of a number.
-    function SignD (a : Double) : Int {
-        if   (a < 0.0) { -1 }
-        elif (a > 0.0) { +1 }
-        else           {  0 }
+    function SignD(a : Double) : Int {
+        if (a < 0.0) { -1 } elif (a > 0.0) { + 1 } else { 0 }
     }
 
     /// # Summary
     /// Returns -1, 0 or +1 that indicates the sign of a number.
-    function SignL (a : BigInt) : Int {
-        if   (a < 0L) { -1 }
-        elif (a > 0L) { +1 }
-        else          {  0 }
+    function SignL(a : BigInt) : Int {
+        if (a < 0L) { -1 } elif (a > 0L) { + 1 } else { 0 }
     }
```

Comment:
For these, I feel like "regular" block formatting with newlines would look better.

Though either option is an improvement over the original formatting.

Reply:
done